### PR TITLE
Update the location, depth and seasons for SOSE transects

### PR DESCRIPTION
**Pull request by xylar**
This merge adds a `'none'` option for `contourLevels` options.  This allows contours to be turned off if they are on by default (in `config.default`), which was not possible previously.

SOSE transects are given more useful longitude values and depths, and contours are turned off by default.  JAS and JFM seasons are added for these transects.  Longitudes passed to the SOSE transects task are now sorted to make sure they don't cause errors when trying to index them later in xarray.

This merge also does a little bit of house cleaning, adding missing pyproj and setuptools packages to the conda recipe and readme.

### CODE REVIEW

**Comment by xylar** [comment]
@milenaveneziani, I will post an example webpage with the new transects as soon as I have time.  All I need from you as a review (besides glancing at the changes) is to let me know if you think the new transects seem useful or if there are changes you would make.

**Comment by xylar** [comment]
addresses #461, #460, #458 and #455

**Comment by milenaveneziani** [comment]
Checked changes, look good to me.

File: mpas_analysis/config.default [hunk 1]
```diff
@@ -1582,7 +1602,8 @@ minLat = -80
 maxLat = -60
 
 # longitudes of transects
-longitudes = numpy.linspace(0, 330, 12)
+# Default transects are at Filchner, Thwaites, Ross, Totten, Amery, Fimbul
+longitudes = [318., 253., 187., 117., 75., 0.]
```
[the remark this answers is not in the document]
Comment (xylar) [reply mpas_analysis/config.default]: Sounds good.  I'll add a transect there.

[the remark this answers is not in the document]
Comment (xylar) [reply mpas_analysis/config.default]: Done!  I tried a few longitudes and 280 did, indeed, seem the best.

**Comment by xylar** [comment]
There are some issues with CI (#463 and #464) that I'm going to try to deal with.  If I can fix them quickly, I'll do that before this PR.  If not, I might just merge this and fix those unrelated issues later.

**Comment by xylar** [comment]
Here is an example of the output with the new set of transects:
http://portal.nersc.gov/project/acme/xylar/sose_transects/GMPAS-IAF.oEC60to30v3wLI.modified_runoff/ocean/

**Comment by xylar** [comment]
I re-tested this with the fixes from #465.  Everything ran fine.

**Comment by xylar** [comment]
@milenaveneziani, are you good with this PR (assuming CI passes okay in a few minutes)?

**Comment by milenaveneziani** [comment]
Yes, this looks great. Please go ahead and merge. Thanks!

**Comment by xylar** [comment]
Thanks, @milenaveneziani!

**Pull request merged**